interface/src/app/gene/resource.js

```diff
@@ -1,22 +1,29 @@
-angular.module('adage.gene.resource', ['ngResource'])
+angular.module('adage.gene.resource', [
+  'ngResource',
+  'adage.utils'
+])
 
-.factory('Gene', ['$resource', function($resource) {
+.factory('Gene', ['$resource', 'ApiBasePath', function($resource, ApiBasePath) {
   return $resource(
-    '/api/v0/gene/:id', {
+    ApiBasePath + 'gene/:id', {
       id: '@id'
     }, {
-
+      post: {
+        url: ApiBasePath + 'gene/',
```

Comment:
Good question. I think if the `:id` stays as it is in the GET then if it does get passed during a POST, the back-end will get tripped up since we're now hitting the "detail" REST endpoint rather than the "list" REST endpoint. This will mean the Tastypie logic we've added into `django-genes` to do the POST back to GET conversion won't get triggered and we'll get an error.

We aren't using the `:id` parameter during POST so I haven't tested that case explicitly, but I'm pretty sure it could foul things up.